ctags/parsers/c.c

```diff
@@ -2234,12 +2232,12 @@ static boolean skipPostArgumentStuff (statementInfo *const st,
 	{
 		switch (c)
 		{
-			case ')':				break;
-			case ':': skipMemIntializerList (token);break;	/* ctor-initializer */
-			case '[': skipToMatch ("[]");			break;
-			case '=': cppUngetc (c); end = TRUE;	break;
-			case '{': cppUngetc (c); end = TRUE;	break;
-			case '}': cppUngetc (c); end = TRUE;	break;
+		case ')':                               break;
+		case ':': skipMemIntializerList (token);break;  /* ctor-initializer */
+		case '[': skipToMatch ("[]");           break;
+		case '=': cppUngetc (c); end = TRUE;    break;
+		case '{': cppUngetc (c); end = TRUE;    break;
+		case '}': cppUngetc (c); end = TRUE;    break;
```

Comment:
> Would it be too hard to do it the other way round and add Vala to uctag's c.c so it supports the same set of languages?

Well, the Vala support in *c.c* has known issues and hacks, so I'm not sure it'd be better to add it in uctags than splitting it up in Geany and add the split version in uctags.